caiman/source_extraction/cnmf/cnmf.py

```diff
@@ -337,7 +342,7 @@ def fit_file(self, motion_correct=False, indices=None, include_eval=False):
             logging.warning("Error: File not found, with file list:\n" + fnames[0])
             raise Exception('File not found!')
 
-        base_name = pathlib.Path(fnames[0]).stem + "_memmap_"
+        base_name = (pathlib.Path(output_dir) / (pathlib.Path(fnames[0]).stem + "_memmap_")).as_posix()
```

Comment:
I'd double check here on how `base_name` is used. I think `base_name` should not contain any path, but only a filename (or prefix to a filename), so we should not join `output_dir` here.

Reply:
@ttngu207  `base_name` is passed only to `mmapping.save_memmap` together with the full path file names in line 369 or 372, depending on a condition. Addition of another path related keyword looks excessive to me. There is only one caveat. The docstring in `mmapping.save_memmap` clearly states that the `base_name` must not contain underscore character. So, as long as this caveat is considered we should be good.